.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!-- To report an issue, fill in all the applicable fields below. -->
+
+#### Platform
+<!-- OS version, library version -->
+
+#### Steps to Reproduce
+1. 
+2.
+
+#### Expected result
+
+#### Observed result
+
+#### Possible cause
+
+#### Related issues
+
```

Comment:
You're right, this assumes it is a bug type issue. It is similar to wavelab/autonomoose#463 

I don't think it is too verbose for a bug type issue. I gave a presentation to WaveLab on this topic: [How to report a bug (if you want it fixed)](https://drive.google.com/open?id=1kLkn-x4H1uTNoWQ-xmQJcRRcd-RyRl3kZl3c9QUfaBQ). The goal is more clear and descriptive bug reports, and such a template could be valuable if we receive bug reports from external users.

For an internal "new feature" type issues we can just clear the form.